src/Command/InfectionCommand.php

```diff
@@ -276,8 +276,8 @@ private function applyMemoryLimitFromPhpUnitProcess(Process $process, PhpUnitAda
     {
         $tempConfigPath = \php_ini_loaded_file();
 
-        if (empty($tempConfigPath) || !file_exists($tempConfigPath)) {
-            // Cannot add a memory limit when there's no php.ini
+        if (empty($tempConfigPath) || !file_exists($tempConfigPath) || !is_writable($tempConfigPath)) {
+            // Cannot add a memory limit when there's no php.ini, or it isn't writeable
```

Comment:
`Cannot add a memory limit: there is no php.ini file or it is not writeable`?

Reply:
Yeah, when we don’t use XDebugHandler, there’s no temporary .ini file, and we can’t write to /etc/.../php.ini

Reply:
Oh. True, that's going to work better, thanks.